### DISCUSSION TITLE
How to reset the input field of a Select component (Multiple and Filterable attributes activated) when clicking on an option?

### DISCUSSION THREAD

**Discussion by Drekstop** [element-plus/element-plus#13528] [Q&A] [answered]
Hi everyone, 

I'm using Vite  + Vue3 Composition API + Script Setup + Typescript. 

I'm trying to reset the input value when an option is clicked.

An exemple usecase with the behaviour I'm looking for: 

- I write "Elem" in the input field of the select component
- The option "Element +" is given
- I click on this option
- The tag "Element +" is added
- The input "Elem" is deleted -> the input field is now blank.

I'm quite new with Vue... I feel that there is a way to access the input variable through decomposing the v-model but not sure about it, i'm still searching. 

Thanks for your help !

**Comment by kaine0923** [comment, marked as answer]
I think you can just use [reserve-keyword](https://element-plus.org/zh-CN/component/select.html#select-attributes) and set it to false

**Reply by Drekstop** [reply]
Hey, thanks a lot, it does work ! It seems I misunderstood the meaning of "reserve" in the attribute description... Have a nice day.
